src/bilder/images/concourse/templates/vector.yaml

```diff
@@ -19,28 +25,61 @@ transforms:
     source: |
       event, err = parse_json(.message)
       if event != null {
-        . = merge(., event)
+        .,err = merge(., event)
         .@timestamp = .timestamp
         .labels = ["concourse", "${ENVIRONMENT}"]
         .environment = "${ENVIRONMENT}"
       }
 
+  add_labels_to_metrics:
+    type: remap
+    inputs:
+    - '*metrics'
+    source: |
+      .tags.environment = "${ENVIRONMENT}"
+      .tags.application = "concourse"
+      .tags.service = "concourse"
+      .tags.hostname = "${HOSTNAME}"
```

Comment:
We don't need to add the hostname tag. That's one of the labels that contributes to our cardinality issues.

Reply:
👍